gatsby-node.js

```diff
@@ -115,6 +119,10 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
         filter: { frontmatter: { published: { eq: true } } }
       ) {
         nodes {
+          frontmatter{
+            program
+            programSlug
```

Comment:
By looking at the code below, I don't find much sense in using 2 different frontamtters for the `programs`, only the value that we set for `programsSlug` can be used and instead use the key `program` for it. It will make the file cleaner as well.

Reply:
The reason I kept two things is that we use the program name to display on the grid. And the programSlug is for the generic url.